packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -120,31 +122,31 @@ class LoadTestDataStoreModel {
     ) {
         await LoadTestDataStoreModel.waitForCatchup(runtime);
 
-        if(!root.hasSubDirectory(config.runId.toString())) {
```

Comment:
all these whitespace changes make the review much more difficult to review, in fact some files only have whitespace changes. would be good to do these in a seperate review if you find them important